integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html

```diff
@@ -0,0 +1,36 @@
+{% extends "../_todo_dashboard_row.html" %}
+{% load i18n %}
+{% block todo_dashboard_icon %}
+    view
+{% endblock todo_dashboard_icon %}
+{% block todo_dashboard_title_link %}
+    {% url 'pages' region_slug=request.region.slug language_slug=default_language_slug %}?status=REVIEW
+{% endblock todo_dashboard_title_link %}
+{% block todo_dashboard_title %}
+    {% translate "Pages that are waiting for review" %}
+{% endblock todo_dashboard_title %}
+{# djlint:off #}
+{% block todo_dashboard_number %}
+    {{ unreviewed_pages|length }}{% endblock todo_dashboard_number %}
+{# djlint:on #}
+{% block todo_dashboard_description %}
+    {% if unreviewed_pages %}
+        {% blocktranslate trimmed %}
+            The page <b>{{ single_unreviewed_page }}</b> is waiting for a review. Please review, if the content on the page is correct.
+        {% endblocktranslate %}
+    {% else %}
+        {% blocktranslate trimmed %}
+            At the moment there is no page waiting for a review. Good job!
+        {% endblocktranslate %}
+    {% endif %}
+{% endblock todo_dashboard_description %}
+{% block todo_dashboard_button_link %}
+    {% if unreviewed_pages %}
+        <a class="btn !rounded-full"
+           href="{% url 'edit_page' region_slug=request.region.slug language_slug=default_language_slug page_id=single_unreviewed_page.page_id %}">
```

Comment:
```suggestion
           href="{% url 'edit_page' region_slug=request.region.slug language_slug=default_language_slug page_id=unreviewed_pages.0.page_id %}">
```
No need to define a new variable just to access the first element